### DISCUSSION TITLE
Website is slow; no immediate action required

### DISCUSSION THREAD

**Discussion by Gomesz785** [OpenTabletDriver/OpenTabletDriver#1833] [General]
I am just informing this situation, not that I am complaining.
If anyone is accessing opentabletdriver.net, wait around 1.5 minutes for the cdn.jsdelivr.net request to load.

**Comment by jamesbt365** [comment]
Nope, basically instant, maybe this was a issue then but not now?

**Reply by Gomesz785** [reply]
Yes, it seems.

I said wait, if it loads early don't worry.
